pkg/apis/scheduling/v1alpha1/types_location.go

```diff
@@ -25,7 +25,8 @@ const (
 	// representation of the location labels in order to use them in a table column in the CLI.
 	LocationLabelsStringAnnotationKey = "scheduling.kcp.io/labels"
 
-	// PlacementAnnotationKey is the label key for the label holding a PlacementAnnotation struct.
+	// PlacementAnnotationKey is the label key for a namespace that indicates the namespace's labels match the selector
+	// in at least one ready Placement.
```

Comment:
this drops the specification of the data format. Has this become a string and not a struct?

Reply:
@davidfestal @qiujian16 @jmprusi can one of you clarify?

Reply:
I looked everywhere and this is the only place "PlacementAnnotation" is mentioned. Also the code only ever sets the empty string and checks for presence.